packages/blockchain-info-components/src/Images/Images.ts

```diff
@@ -14,6 +14,8 @@ import allowCamera3 from './img/allow-camera@3x.png'
 import appleAppStoreBadge from './img/apple-app-store-badge.svg'
 import applePay from './img/apple-pay-icon.svg'
 import arrowLeft from './img/arrow-left.svg'
+import badgeAndroidPlaystore from './img/badge-android-playstore.svg'
+import badgeIosAppstore from './img/badge-ios-appstore.svg'
```

Comment:
i believe we already have these in blockchain-info-components/src/Badges

Reply:
this is done! They are not quite the same images but they'll do their job. Thanks andrew.